samples: dect_phy: dect_shell: new startup_cmd and possibility to give AT commands from command prompt (#18548)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

    * The ``dect mac`` command.
      A brief MAC-level sample on top of DECT PHY interface with new commands to create a periodic cluster beacon, scan for it, associate or disassociate a PT/client, and send data to a FT/beacon random access window.
      This is not a full MAC implementation and not fully compliant with DECT NR+ MAC specification (`ETSI TS 103 636-4`_).
    * The ``startup_cmd`` command.

>       New command to store shell commands to be run sequentially after a bootup.

Reviewer: ```suggestion
      This command is used to store shell commands to be run sequentially after bootup.
```
